js/tests/unit/button.js

```diff
@@ -60,6 +60,19 @@ $(function () {
     assert.strictEqual($btn.attr('aria-pressed'), 'true', 'btn aria-pressed state is true')
   })
 
+  QUnit.test('should toggle aria-pressed on buttons with container', function (assert) {
```

Comment:
@patrickhlauke @Johann-S this is due to Hound not supporting the cascading ESLint configs we use. Hound was recently set to ignore our test files, until a workaround can be found. Travis lints everything correctly.